i18n/zh-CN/docusaurus-plugin-content-blog/2022-02-21-linkis-deploy.md

```diff
@@ -208,12 +208,21 @@ YARN_RESTFUL_URL=http://xx.xx.xx.xx:8088
 ```
 
 #### 引擎版本信息 
+如果spark不是2.4.3的版本，需要修改参数：
 ```shell script
-##如果spark不是2.4.3的版本(可以在linkis-package/lib/linkis-engineconn-plugins/hive/plugin中查看引擎版本)，需要修改参数：
-#SPARK_VERSION=3.1.1
-
-##如果hive不是2.3.3的版本，需要修改参数：
-#HIVE_VERSION=2.3.4
+## Engine version conf
+#SPARK_VERSION，如果安装的Spark版本不是2.4.3，则需要修改为相应的版本，如3.1.1
+SPARK_VERSION=3.1.1
+##HIVE_VERSION
```

Comment:
![image](https://user-images.githubusercontent.com/7869972/172819000-048c5cb0-ff80-4f9c-8bee-989af921c891.png)
redundant information can be removed

Reply:
Done